.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,103 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  publish-release-package-cache:
+    runs-on: ubuntu-4-cores-150GB-ssd
+    container:
+      image: asterinas/asterinas:0.16.2-20251209
+      options: -v /dev:/dev --privileged
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build Asterinas NixOS patched packages and push to release cache
+        run: |
+          make kernel BOOT_PROTOCOL=linux-efi-handover64 || true
+          export CACHIX_AUTH_TOKEN=${{ secrets.CACHIX_AUTH_TOKEN_FOR_RELEASE_CACHE }}
+          make push_cachix USE_RELEASE_CACHE=1 2>&1 | tee cachix.log || true
+          tail --lines 10 cachix.log | grep -q -E "^(All done|Nothing to push)" || (echo "Push cachix failed" && exit 1)
+          echo "Push cachix succeeds!"
+
+  create-release:
+    runs-on: ubuntu-latest
+    needs: [ publish-release-package-cache ]
+    outputs:
+      tag: ${{ steps.tag.outputs.tag }}
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get Asterinas version
+        id: tag
+        run: |
+          ASTER_VERSION=$(cat ./VERSION)
+          echo "ASTER_VERSION=${ASTER_VERSION}"
+          echo "tag=${ASTER_VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Create release
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_REPO: ${{ github.repository }}
+        run: |
+          tag="${{ steps.tag.outputs.tag }}"
+          body="asterinas-nixos/asterinas@$tag"
```

Comment:
This field is the body of the Github Release, right? If so, we can make it more human readable:

```
body="Write release notes for Asterinas $tag"
```